### ISSUE TITLE
Dual boot

### ISSUE THREAD

**Issue by r0n121** [Alex313031/ThoriumOS#44]
Is there a way to dual boot thorium os ?

**Comment by Alex313031** [comment]
@r0n121 Yes, but it isn't super easy. And it also depends on what OS you are wanting to dual boot with. You can install it, and then mount the drive on a Linux machine and shrink partition 1 (the "Stateful" partition), and then install a Linux distro onto the free space that is created. However, dual booting windows is more complicated, as you have to do the above, as well as reinstall the ChromiumOS bootloader, and add an entry to boot the Windows partition. Unfortunately, while FydeOS has much of their stuff open source, their code they are using to allow easy dual boot via the GUI during initial installation is not open source, and I have no idea how to implement it myself.
